README.md

```diff
@@ -38,6 +45,14 @@ Brion provides the following major features:
   [Lunchbox](http://eyescale.github.io/Lunchbox-1.8/index.html),
   [Boost](http://www.boost.org/doc/libs).
 
+## Algorithmic library
```

Comment:
Could be later on, but not really at this moment. I was waiting for some input before rewriting it because I wasn't convinced either.